pkg/rancher-desktop/components/Snapshots.vue

```diff
@@ -87,15 +87,15 @@ export default Vue.extend<Data, Methods, Computed, never>({
     <Banner
       v-if="snapshotEvent"
       class="banner mb-20"
-      color="success"
+      :color="snapshotEvent.result"
       :closable="true"
       @close="snapshotEvent=null"
     >
-      <span v-html="t(`snapshots.info.${ snapshotEvent.type }.${ snapshotEvent.result }`, { snapshot: snapshotEvent.snapshotName }, true)" />
+      <span v-html="t(`snapshots.info.${ snapshotEvent.type }.${ snapshotEvent.result }`, { snapshot: snapshotEvent.snapshotName, error: snapshotEvent.error }, true)" />
     </Banner>
     <div
-      v-for="item of snapshots"
-      :key="item.id"
+      v-for="(item, idx) of snapshots"
+      :key="idx"
```

Comment:
It looks like the snapshot name would better serve as a key in this scenario.

```suggestion
      v-for="(item) of snapshots"
      :key="item.name"
```

Using the index as a key is rarely the correct choice when working with v-for; indexes are not unique identifiers for an array and adding/removing items can lead to undefined behaviors.